example/App.js

```diff
@@ -25,13 +26,24 @@ import TakeSnapshot from './examples/TakeSnapshot';
 import FitToSuppliedMarkers from './examples/FitToSuppliedMarkers';
 import StaticMap from './examples/StaticMap';
 
+function makeExampleMapper(useGoogleMaps) {
+  if (useGoogleMaps) {
+    return example => [
+      example[0],
+      [example[1], example[3]].filter(Boolean).join(' '),
+    ];
+  }
+  return example => example;
+}
+
 class App extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
       Component: null,
-      useGoogleMaps: false,
+      showGoogleMapsSwitch: Platform.OS === 'ios',
```

Comment:
Not a big deal, but this doesn't have to be in `this.state` because it never changes.